templates/core/terraform/appgateway/appgateway.tf

```diff
@@ -14,7 +14,7 @@ resource "azurerm_application_gateway" "agw" {
   sku {
     name     = "Standard_v2"
     tier     = "Standard_v2"
-    capacity = 2
+    capacity = 1
```

Comment:
Is this for the docker container? or if not, can you mention why this change is made?

Reply:
Ah, not really part of this PR, we only need 1 - half the cost - can put it back or in another PR if think I should.

Reply:
No - it's cool, was just curious :)